README.md

```diff
@@ -60,6 +60,7 @@ A lightweight and fast control to render a select component that can display hie
     - [always](#always)
   - [form states (disabled|readOnly)](#formstates)
   - [id](#id)
+  - [searchPredicate](#searchPredicate)
```

Comment:
This needs to be all lowercase like `(#searchpredicate)`. GH doesn't link otherwise. Gets me everytime. :)

But don't submit another PR. I'll be fixing this in a doc update. Experimenting with a new readme layout that doesn't need so much scrolling.

Something like

![image](https://user-images.githubusercontent.com/781818/60385903-ef104380-9a5c-11e9-9a55-cf1c449ce7e0.png)